liota/core/package_manager.py

```diff
@@ -426,9 +448,15 @@ def run(self):
                         continue
                     file_name = msg[1]
                     if command == "unload":
-                        self._package_unload(file_name)
+                        if (self._package_unload(file_name) == True):
+                            print "succeeded to unload:", file_name
+                        else:
+                            print "failed to unload:", file_name
                     elif command == "delete":
```

Comment:
Does this delete the package?

Reply:
Yes, the "delete" cmd will delete the package in file system.